internal/guestpath/paths.go

```diff
@@ -0,0 +1,57 @@
+package guestpath
+
+import (
+	"path/filepath"
+	"runtime"
+)
+
+const (
+	// LCOWRootPrefixInUVM is the path inside UVM where LCOW container's root file system will be mounted
+	LCOWRootPrefixInUVM = "/run/gcs/c"
```

Comment:
A few of these don't appear to be used outside this package, suggest we don't export them.

Reply:
They were used in this PR https://github.com/microsoft/hcsshim/pull/1258, so I left them like that to avoid conflicts.

Reply:
Since that PR is in now, can you rebase so they don't show up in this diff?

Reply:
I did rebase. I formatted the comment lengths in this file and now they show up in the diff 😄